components/resources/wallet_strings.grdp

```diff
@@ -9,6 +9,7 @@
   <message name="IDS_WALLET_EXPECTED_SINGLE_PARAMETER" desc="The text of the response for wallet_addEthereumChain call with empty parameters">Expected single, object parameter.</message>
   <message name="IDS_WALLET_CHAIN_EXISTS" desc="The text of the response for wallet_addEthereumChain call with existing chain id">This Chain ID is currently used</message>
   <message name="IDS_WALLET_INTERNAL_ERROR" desc="The text of the response for wallet_addEthereumChain call with internal error">An internal error has occurred</message>
+  <message name="IDS_WALLET_INVALID_BLOCKHASH_ERROR" desc="The text of the response for Solana signTransaction and signAllTransactions with invalid blockhash or unable to validate due to network or RPC error">Blockhash is invalid or unable to be validated</message>
```

Comment:
Suggestion: Blockhash is invalid or can not be validated

Reply:
Thanks, updated as suggested.